common.js

```diff
@@ -1,10 +1,13 @@
+'use strict'
+
```

Comment:
It's more of a consistency and standards issue. You're only putting it in *some* of the files. In terms of standards, I found some interesting GitHub issues around it, but it's irrelevant to this PR.